src/assets/feature-flags.json

```diff
@@ -265,6 +265,10 @@
               {
                 "id": "varsSecrets",
                 "developmentState": "stable"
+              },
+              {
+                "id": "webhooks",
+                "developmentState": "stable"
```

Comment:
I think you've introduced several elements in UI, which would be disabled for now. They need their feature flags.

Reply:
Added

```
 {
                "id": "webhooks",
                "developmentState": "stable",
                "subfeatures": [
                  {
                    "id": "viewDeliveryReport",
                    "developmentState": "mockup"
                  },
                  {
                    "id": "rotateSecret",
                    "developmentState": "mockup"
                  },
                  {
                    "id": "verifyNow",
                    "developmentState": "mockup"
                  }
                ]
              }
```